core/src/test/java/org/elasticsearch/bwcompat/RestoreBackwardsCompatIT.java

```diff
@@ -98,7 +98,7 @@ public void testRestoreOldSnapshots() throws Exception {
         for (Version v : VersionUtils.allReleasedVersions()) {
             if (VersionUtils.isSnapshot(v)) continue;  // snapshots are unreleased, so there is no backcompat yet
             if (v.isRelease() == false) continue; // no guarantees for prereleases
-            if (v.onOrBefore(Version.V_2_0_0_beta1)) continue; // we can only test back one major lucene version
+            if (v.onOrBefore(Version.V_5_0_0_beta1)) continue; // we can only test back one major lucene version
```

Comment:
Wondering - we didn't you go with `Version.CURRENT.minimumIndexCompatibilityVersion()` here?